Add README.md

vipulgupta2048 opened the pull request:
The documentation of this activity has been migrated from the wiki. 
Check for more information godiard/help-activity/pull/43

This PR adds a basic README where 'How to use Ruler' points to the migrated user-documentation.

Feel free to add/subtract information as necessary from the README.md file

Status - Merge after godiard/help-activity/pull/43 gets merged

quozl commented:
Reviewed to 2ab156a.

* [x] copy and paste error; dangling reference to Turtle Confusion,
* [x] commit message is very brief and does not use imperative mood, see [making commits](https://github.com/sugarlabs/sugar-docs/blob/master/src/contributing.md#making-commits).
* [ ] activity fails to work, per https://github.com/sugarlabs/ruler/issues/4, https://github.com/sugarlabs/ruler/issues/3, and now https://github.com/sugarlabs/ruler/issues/6, so time invested in a README.md seems wasted.

vipulgupta2048 commented:
- Fixed the reference, fixed some more things in README.md 
- Commit message amended  
- I wouldn't call it wasted :smile: mistakes teach like these teach me to be more considerate and not rush into things. I should have completely understood the state of the activity before implementing the migration script. Also, this finds me my new tasks to work and contribute on. I request the PR to remain open until the issues are resolved as mentioned in https://github.com/godiard/help-activity/pull/43
Thanks

quozl commented:
Thanks.
